docs/sources/operations/recording-rules.md

```diff
@@ -78,6 +78,13 @@ Remote-write can be tuned if the default configuration is insufficient (see [Fai
 
 There is a [guide](https://prometheus.io/docs/practices/remote_write/) on the Prometheus website, all of which applies to Loki, too.
 
+Rules can be evenly distributed across available rulers by using `-ruler.enable-sharding=true` and `-ruler.sharding-strategy="by-rule"`.
```

Comment:
Good questions put by @kavirajk here. I want to add one more: What is the impact on the ruler WAL when changing between this modes?

Reply:
There is no mechanism for ensuring "exactly once evaluation" of rules within a given interval, which is fine because those metrics will be aggregated by the receiving metrics backend.

As for the WAL: I don't think there will be any impact; any samples already appended to the WAL will be submitted when it is replayed.